python/CMakeLists.txt

```diff
@@ -74,6 +74,9 @@ include(SetupCxxFlags)
 # Add common flags
 set(CMAKE_CXX_FLAGS "${CXX_COMMON_FLAGS} ${CMAKE_CXX_FLAGS}")
 
+# Enable perf and other tools to work properly
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fno-omit-frame-pointer")
```

Comment:
See discussion on https://issues.cloudera.org/browse/IMPALA-4132 — if the performance penality is < 1%, then having it always on seems OK to me for easier perf debugging and core dumps. Kudu (also very performance sensitive) leaves this in too https://github.com/apache/kudu/blob/master/CMakeLists.txt#L176

I'm OK with removing it too